spec/order_spec.rb

```diff
@@ -0,0 +1,32 @@
+require 'order'
+
+describe Order do
+  let(:menu) { Menu.new }
+  let(:dish) { { name:'Burger', price: 8 } }
```

Comment:
I wonder if this could be a double rather than a variable?

Reply:
Good question. When we review the code let's give it a go and see what happens!